citar-org-roam.el

```diff
@@ -40,6 +40,17 @@
   :group 'citar-org-roam
   :type 'string)
 
+(defcustom citar-org-roam-capture-template-key
+  nil
+  "When non-nil, `citar-org-roam--create-capture-note' will use the
+template associated with the key in `org-roam-capture-templates'.
+
+When nil (the default), the template will be creating a org file
```

Comment:
```suggestion
When nil (the default), the template will be creating an org file
```

Reply:
You may be interested in this @lazzalazza?

Reply:
Thanks! I fixed the typo.